clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -627,11 +627,11 @@
   },
   "dependencies": {
     "@code4z/cobol-dialect-api": "file:../cobol-dialect-api",
-    "@vscode/extension-telemetry": "^0.8.2",
+    "@vscode/extension-telemetry": "^0.9.7",
     "glob": "10.3.10",
     "iconv-lite": "^0.6.3",
     "micromatch": "^4.0.7",
-    "vscode-languageclient": "8.0.2"
+    "vscode-languageclient": "^8.1.0"
```

Comment:
- [ ] Is this safe? would this lead to any backward in compatibility for some users ?

Reply:
I guess we will not face with a situation when backend is old and frontend is new. So far we do not have any such customers who are using our code to create their own extensions, so I think we are safe